apis/v1alpha1/httproute_types.go

```diff
@@ -268,7 +288,7 @@ type HTTPRouteMatch struct {
 	// Support: custom
 	//
 	// +optional
-	ExtensionRef *RouteMatchExtensionObjectReference `json:"extensionRef"`
+	ExtensionRef *LocalObjectReference `json:"extensionRef"`
```

Comment:
Add `omitempty` tag if field is optional.

Reply:
Updated, thanks!